src/java/edu/slu/action/ObjectAction.java

```diff
@@ -846,16 +897,18 @@ public void updateObject() throws IOException, ServletException, Exception{
     
     /**
      * A helper function that determines whether or not an object has been flagged as deleted.
+     * This should only be fed reliable objects from mongo
      * @param obj
      * @return A boolean representing the truth.
      */
-    public boolean checkIfDeleted(JSONObject obj){
+    private boolean checkIfDeleted(JSONObject obj){
         boolean deleted = obj.containsKey("__deleted");
         return deleted;
     }
     
     /**
-     * A helper function that gathers an object by its id and determines whether or not it is flagged as deleted.
+     * A public facing servlet to determine if a given object ID is for a deleted object.
```

Comment:
I don't think we need this. If they just get the object it'll be `__deleted`

Reply:
yeah but it is a convenience servlet we can offer for someone who doesn't want to pull the object down and check